### DISCUSSION TITLE
Strategy if substream frame rate is hard-coded at 20 fps?

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#10144] [General]
I have a reolink trackmix that has two lenses and therefore 4 streams: a wide-angle main and sub, as well as a telephoto main and sub. I am setting it up in frigate as two separate cameras. The parameters of the telephoto streams are hard-coded and cannot be configured. The sub stream is fixed at 896x512 resolution and 20 fps. Should I just leave it like that for frigate's detect or am I better off transcoding it first?

**Comment by NickM-27** [comment]
Detect fps doesn't need to match the cameras fps. Just leave detect fps as default
